protocol/synthetix/storage.dump.sol

```diff
@@ -396,7 +395,7 @@ library Account {
         AccountRBAC.Data rbac;
         uint64 lastInteraction;
         uint64 __slotAvailableForFutureUse;
-        uint128 __slot2AvailableForFutureUse;
+        uint128 currentDelegationIntentsEpoch;
```

Comment:
because of how the storage alignment works. `lastInteraction` being an `uint64`, if the next item is a `uint256` it will need to pad it to keep the alignment (and keep the rest unused). That gives us empty slots that can be reused later and why we had 
```
        uint64 lastInteraction;
        uint64 __slotAvailableForFutureUse;
        uint128 __slot2AvailableForFutureUse;
```
filling the 256 bits